po/it.po

```diff
@@ -2,70 +2,71 @@
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the com.github.ryonakano.konbucase package.
 # Mirko Brombin <send@mirko.pm>, 2020.
+# Saverio <saverio.morelli@outlook.it>, 2021.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: com.github.ryonakano.konbucase\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2021-04-13 04:37+0000\n"
-"PO-Revision-Date: 2020-04-24 15:22+0000\n"
-"Last-Translator: Mirko Brombin <send@mirko.pm>\n"
-"Language-Team: Italian\n"
+"PO-Revision-Date: 2021-07-07 11:55+0200\n"
+"Last-Translator: Saverio <saverio.morelli@outlook.it>\n"
+"Language-Team: Italian <saverio.morelli@outlook.it>\n"
 "Language: it\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=(n > 1);\n"
-"X-Generator: Poedit 2.0.6\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"X-Generator: Gtranslator 40.0\n"
 
 #: src/Widgets/ComboEntry.vala:51
 msgid "Space separated"
 msgstr "Spazi separati"
 
 #: src/Widgets/ComboEntry.vala:79
 msgid "Copy to Clipboard"
-msgstr "Copia nella Clipboard"
+msgstr "Copia negli appunti"
 
 #: src/Widgets/ComboEntry.vala:162
 msgid "Each word is separated by a space"
-msgstr ""
+msgstr "Ogni parola è separata da uno spazio"
 
 #: src/Widgets/ComboEntry.vala:164
 msgid "The first character of compound words is in lowercase"
-msgstr ""
+msgstr "Il primo carattere delle parole composte è minuscolo"
 
 #: src/Widgets/ComboEntry.vala:166
 msgid "The first character of compound words is in uppercase"
-msgstr ""
+msgstr "Il primo carattere delle parole composte è maiuscolo"
 
 #: src/Widgets/ComboEntry.vala:168
 msgid "Each word is separated by an underscore"
-msgstr ""
+msgstr "Ogni parola è separata da un underscore (trattino basso)"
 
 #: src/Widgets/ComboEntry.vala:170
 msgid "Each word is separated by a hyphen"
-msgstr ""
+msgstr "Ogni parola è separata da un hyphen (trattino alto)"
 
 #: src/Widgets/ComboEntry.vala:172
 msgid "The first character of the first word in the sentence is in uppercase"
-msgstr ""
+msgstr "Il primo carattere della prima parola della frase è in maiuscolo"
 
 #: src/MainWindow.vala:53
 msgid "Light background"
-msgstr ""
+msgstr "Sfondo chiaro"
 
 #: src/MainWindow.vala:54
 msgid "Dark background"
-msgstr ""
+msgstr "Sfondo scuro"
 
 #. TRANSLATORS: Whether to follow system's dark style settings
 #: src/MainWindow.vala:59
 msgid "Follow system style:"
-msgstr ""
+msgstr "Segui lo stile del sistema"
```

Comment:
```suggestion
msgstr "Segui lo stile del sistema:"
```

Seems like the trailing colon is missing, ignore my comment if it's intentional though.

Reply:
Oh, yes, sorry. My mistake :smile: